internal/sync/util.go

```diff
@@ -20,7 +20,7 @@ func autoConfigure(object any, config config.CustomMap) {
 					reflect.Int16,
 					reflect.Int32,
 					reflect.Int64:
-					field.SetInt(value.(int64))
+					field.SetInt(int64(value.(int)))
```

Comment:
I've just dug a bit through the yaml library. If I've read things correctly, then a numerical value uses `int` if the number is small enough to fit in there, and falls back to `int64` otherwise (https://github.com/go-yaml/yaml/blob/f6f7691b1fdeb513f56608cd2c32c51f8194bf51/resolve.go#L194).

We should also handle that case:
```go
if v, ok := value.(int) ; ok {
field.SetInt(int64(v))
} else {
field.SetInt(value.(int64))
}
```

Reply:
phew, good catch. added :)